src/catalogue/items/catalogueItemsDialog.component.test.tsx

```diff
@@ -621,6 +622,27 @@ describe('Catalogue Items Dialog', () => {
     expect(onClose).not.toHaveBeenCalled();
   }, 10000);
 
+  it('displays recently added section', async () => {
+    vi.setSystemTime(new Date("2024-01-09T12:00:00.000+00:00"));
```

Comment:
```suggestion
    vi.useFakeTimers()
    vi.setSystemTime(new Date("2024-01-09T12:00:00.000+00:00"));
```

From this documentation (https://vitest.dev/guide/mocking.html#example) is looks like you should be using `vi.useFakeTimers()` before running the test.

Reply:
If its only in this test I think this is fine rather than mocking the time for each test when we don't need to.

Reply:
I tried using useFakeTimers before, and was having difficulties advancing time around an await function, however I was able to fix the issue now.

Also, I have added useRealTimers in the afterEach section of the test file.